src/gui/src/PatternEditor/PatternEditorPanel.cpp

```diff
@@ -107,23 +108,20 @@ PatternEditorPanel::PatternEditorPanel( QWidget *pParent )
 //wolke some background images back_size_res
 	PixmapWidget *pSizeResol = new PixmapWidget( nullptr );
 	pSizeResol->setFixedSize( 200, 20 );
-	pSizeResol->setPixmap( "/patternEditor/background_res-new.png" );
+	pSizeResol->setPixmap( "/patternEditor/background_res-new4.png" );
```

Comment:
Probably better to replace the existing asset rather than add a new one while keeping the old one unused.

Reply:
Ok with that, I saw other background image in the img directory that are not used anymore as well.
For example in `data/img/gray/instrumentEditor/` there are `layerTab.png`, `layerTabnew.png `and `layerTabsupernew.png`

Reply:
Hah yes, we should probably clean those up at some point! :D